bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/obscomp/SPSiteQuality.java

```diff
@@ -399,12 +349,14 @@ public String displayValue() {
             return _displayValue;
         }
 
+        public double magAdjustment() {
+            return _magAdjustment;
+        }
+
```

Comment:
That is one option, it used to be kind of peppered in a few different ways. At least in this manner, it is consistent for all conditions